spec/integrations.yml

```diff
@@ -5,36 +5,100 @@
   mutation_coverage: false
   mutation_generation: true
   expect_coverage: 0 # not run
-  exclude:
-    # Binary encoded source subjected to limitations see README of unparser
-    - core/array/pack/{b,h,u}_spec.rb
+  expected_errors:
+    "#<Parser::SyntaxError: invalid multibyte escape: /\xAA/>":
     - language/regexp/escapes_spec.rb
-    - core/array/pack/shared/float.rb
-    - core/array/pack/shared/integer.rb
-    - core/array/pack/{c,m,w}_spec.rb
-    - core/regexp/shared/quote.rb
-    - core/encoding/compatible_spec.rb
-    - core/io/readpartial_spec.rb
-    - core/env/element_reference_spec.rb
-    - core/dir/pwd_spec.rb
-    - core/string/casecmp_spec.rb
-    - core/string/unpack/{b,c,h,m,u,w}_spec.rb
-    - core/string/unpack/b_spec.rb
-    - core/string/unpack/shared/float.rb
-    - core/string/unpack/shared/integer.rb
+    "#<Parser::SyntaxError: literal contains escape sequences incompatible with UTF-8>":
+    - core/array/fixtures/encoded_strings.rb
```

Comment:
So these paths are related to the error string above? It wasn't clear to me that they were related. Before looking closely I thought this was just a list of expected errors.

Would it be syntactically valid to indent each list underneath the error they relate to? I think that might show the relationship better.